bin/generate_practice_exercise.sh

```diff
@@ -20,7 +20,7 @@ echo "Fetching latest version of configlet..."
 echo "Adding instructions and configuration files..."
 UUID=$(bin/configlet uuid)
 jq --arg slug "$SLUG" --arg uuid "$UUID" \
-    '.exercises.practice += [{slug: $slug, name: $slug, uuid: $uuid, practices: [], prerequisites: [], difficulty: 5}]' \
```

Comment:
Is this to avoid a casing type problem with the title of the exercise?

Reply:
Exactly, it's too easy to miss when the name is prefilled, it happened a lot that people didn't change it (me included). At least with a big TODO you can't miss it (hopefully).